.github/workflows/ci.yml

```diff
@@ -109,6 +109,18 @@ jobs:
           make BUILD=build-debug BUILD_TYPE=Debug sdk
           ccache -s
 
+      - name: Strip executables - Linux/Windows
+        if: runner.os != 'macOS' && (github.event_name == 'release' || inputs.strip == 'true')
+        run: |
+          strip -s build/host/sdk/bin/*
+
+      - name: Strip tools - Linux
```

Comment:
Maybe just drop all the stripping of these on all platforms?

Reply:
done.